src/fetch/rewards.py

```diff
@@ -0,0 +1,77 @@
+"""Functionality for rewards."""
+
+from fractions import Fraction
+
+import pandas as pd
+from pandas import DataFrame
+
+from src.config import RewardConfig
+from src.logger import set_log
+
+log = set_log(__name__)
+
+BATCH_REWARDS_COLUMNS = ["solver", "primary_reward_eth"]
+QUOTE_REWARDS_COLUMNS = ["solver", "num_quotes"]
+
+REWARDS_COLUMNS = [
+    "solver",
+    "primary_reward_eth",
+    "primary_reward_cow",
+    "quote_reward_cow",
+    "reward_token_address",
+]
+
+
+def compute_rewards(
```

Comment:
I was thinking whether we should express rewards in their "native currency", i.e., batch rewards in native token and quote rewards in COW, and then at the final step, when we build the payout, actually do the conversions.